.env.default

```diff
@@ -24,4 +24,7 @@ GITHUB_GITHUB_REDIRECT_URI=
 
 # Data importer Github login
 GITHUB_USERNAME=
-GITHUB_PERSONAL_TOKEN=
\ No newline at end of file
+GITHUB_PERSONAL_TOKEN=
+
+# CORS allowed origins
+SKOSMOS_URI=
```

Comment:
tarviiko tätä muuten asettaa näin lokaalidevaajan näkökulmasta jos ei skosmosta pyörittele? eli tää pitää vaan tonne testi/tuotantopalvelimille muistaa asettaa?

Reply:
ei tarvitse tosiaan – ainoastaan jos haluaa luoda uusia ehdotuksia tämän järjestelmän ulkopuolelta niin tarvitsee tämän